5.js

```diff
@@ -0,0 +1,19 @@
+const disemvowel = string => arrayToString(filterOutSpaces(filterOutVowels(stringToArray(string))))
+
+const stringToArray = string => string.split('')
+
+const arrayToString = array => array.join('')
+
+const filterOutVowels = characters => characters.filter(
+  character => character !== 'a'
+            && character !== 'a'
+            && character !== 'i'
+            && character !== 'o'
+            && character !== 'u'
+)
+
+const filterOutSpaces = characters => characters.filter(
+  character => character !== ' '
+)
+
+module.exports = disemvowel
```

Comment:
Can use filter or reduce with regex to accomplish this without having to change to an array and then back in to a string. That said, this works and is creative so good work.